src/Compilers/CSharp/Portable/Emitter/EditAndContinue/PEDeltaAssemblyBuilder.cs

```diff
@@ -55,7 +55,7 @@ internal sealed class PEDeltaAssemblyBuilder : PEAssemblyBuilderBase, IPEDeltaAs
                 RoslynDebug.AssertNotNull(previousGeneration.PEModuleBuilder);
 
                 var previousAssembly = ((CSharpCompilation)previousGeneration.Compilation).SourceAssembly;
-                var previousContext = new EmitContext((PEModuleBuilder)previousGeneration.PEModuleBuilder, null, new DiagnosticBag(), metadataOnly: false, includePrivateMembers: true);
+                var previousContext = new EmitContext((PEModuleBuilder)previousGeneration.PEModuleBuilder, null, null, new DiagnosticBag(), metadataOnly: false, includePrivateMembers: true);
```

Comment:
nit: it would be nice to label these null literals the same way we label the bool literals